Fix AudioSolver and Reimplemented ImageSolver

MacKey-255 opened the pull request:
Added the human factor in the SolverAudio.
Improved speech system in the SolverAudio.
"Try again later" message detection in SolverAudio.
WitAI service added.
Secundary_speech_service was added to avoid errors of the first voice service in SolverAudio.
Simple system of proxies added through the json file.
Added selector of the method to solve the captcha.
Better error handling (Avoid connection errors and other common ones).
Shipping button implemented (Solve captcha and submit form).
Improved information provided by the records.
Implementing the Yolov3 neural network in the SolverImage system.
Reimplemented the SolverImage system.
Neural network without training (default on Yolo v3)
The network must be trained, I don't have a computer to do that.
Prediction only to ('bus', 'car', 'bicycle', 'fire hydrant', 'traffic light', 'vehicles', 'motorcycle')
Auto Download weight yolo v3 not testing (My connection is slow)

Commit:
Fix AudioSolver and Reimplemented ImageSolver

Added the human factor in the SolverAudio.
Improved speech system in the SolverAudio.
"Try again later" message detection in SolverAudio.
WitAI service added.
Secundary_speech_service was added to avoid errors of the first voice service in SolverAudio.
Simple system of proxies added through the json file.
Added selector of the method to solve the captcha.
Better error handling (Avoid connection errors and other common ones).
Shipping button implemented (Solve captcha and submit form).
Improved information provided by the records.
Implementing the Yolov3 neural network in the SolverImage system.
Reimplemented the SolverImage system.
Neural network without training (default on Yolo v3)
The network must be trained, I don't have a computer to do that.
Prediction only to ('bus', 'car', 'bicycle', 'fire hydrant', 'traffic light', 'vehicles', 'motorcycle')
Auto Download weight yolo v3 not testing (My connection is slow)

Commit:
Add new titles

Fix predict awaited
Add new titles in spanish

opclient commented:
![image](https://user-images.githubusercontent.com/30980283/62082765-9ad8c900-b27f-11e9-983a-eaf3d0cd8ebc.png)

MacKey-255 commented:
Men, Remember that you are using a proxy so the response time is slower. Try to time out set to 300 seconds. SAlu2s

opclient commented:
I try using localhost

MacKey-255 commented:
Men, try using it without a proxy and you'll see how it works. I tested it on a dedicated server and on my PC and everything is on wheels, the only thing missing is to train the neural network. SAlu2s

Commit:
Fix Select Image detection

Add detection if image one selected

Commit:
Update my Fork from Francesco149/nonoCAPTCHA

Upgrade for Windows thanks for Francesco149
Upgrade Predict Download Model yolov3

Commit:
Rename Project

Rename project to GoodByeCaptcha
Fix Merge with Francesco149

Commit:
Update README.rst

Commit:
Update all Requeriments

Commit:
Upload projecto to Pypi

Commit:
Upgrade version

Commit:
Downgrade Requirements

Downgrade requirements because incompatibility with code

MacKey-255 committed:
Update README.rst

MacKey-255 committed:
Delete Proxy and More changes

Delete simple JSON PROXY
Set random proxy option
Add .gitignore my publish.sh
Remove launcher.py
Fix image solver 40%

MacKey-255 committed:
Merge remote-tracking branch 'origin/master'

MacKey-255 committed:
Fix image solver

Restructurado image solver
Update README.rst
Change library to Production/Stable

MacKey-255 committed:
Update README

Change links pypi

MacKey-255 committed:
Update version

MacKey-255 committed:
Add events

Add event before and after start solver
Add event before goto page
Image solver Full Tested

MacKey-255 committed:
Release 2.3.1

Ready release

MacKey-255 committed:
Add models folder

Add models folder to pypi

MacKey-255 committed:
Testing Pypi deploy

Add verify tmp dir
Add comment on examples

MacKey-255 committed:
Add extra plugins

Add extra pyppeteer plugins

MacKey-255 committed:
Update README.rst

Add video tutorial "How to install goodbyecaptcha"

MacKey-255 committed:
Add description training yolov3

MacKey-255 committed:
Restructure project

Restructure Base class
Add more comments to code
Add more functionalities to Base class
Optimize and reformat code
Restructure project not complete

MacKey-255 committed:
Optimize Image Solver

Optimize, clearup and Reformat code
Add new config and delete obsolete
Remove unnecessary files
Add more comments in Images Solver
Change Wait Iframe for wait full load page
Add retry in get_page
Update requirements (No complete)
Add UTF code to README
Adjust examples (No all tested)
Go to Audio Solver :)

MacKey-255 committed:
Optimize Audio Solver

Move code of Audio to audio.py
Adjust Code to Base
Remove block_images (Error load Checkbox)
Add Exception  => ResolveMoreLater
Clearup and reformat code

MacKey-255 committed:
Change to version 2.4.0

No tested All :|

MacKey-255 committed:
Testing Audio Solver 1/2

Add Incognito Page
Remove Site-Key (Not use for now)
Adjust logging in Base Project
Add FakeBrowser (Bypass Bot detections)
Testing Audio method 1/2 ( Try Later :V )
Add Try Catch to Try Again Message

MacKey-255 committed:
Testing Audio Solver 2/2

Tested Audio Solver
Add Set Language
Tested all Examples

MacKey-255 committed:
Add SourceForge Downloads

In the next ... migrate to Yolo V5 for to better detect images

MacKey-255 committed:
Update Requirements

MacKey-255 committed:
Fix Chrome Error in Pyppeteer

Add ChromePath how parameters. Fix: https://github.com/pyppeteer/pyppeteer/issues/108

MacKey-255 committed:
Fix issues

Add args on Browser
Add verify_ssl False on UserAgent
Update Setup.py & Version

Helias committed:
docs(readme): add python3-pocketsphinx as requirement

Helias committed:
Update README.rst

MacKey-255 committed:
Merge pull request #12 from Helias/patch-1

docs(readme): add python3-pocketsphinx and libpulse-dev as requirements

Jourdelune committed:
Update requirements.txt

MacKey-255 committed:
Merge pull request #18 from Jourdelune/master

Change requirements of opencv-python

OTHFamily committed:
Update requirements.txt

MacKey-255 committed:
Merge pull request #22 from OTHFamily/patch-2

Update requirements.txt
